utilities/durbinlab/README.md

```diff
@@ -17,3 +17,10 @@ We suggest using [`rclone`](https://rclone.org) to assist with data uploads to t
 
    - Please note: many rclone commands are recursive __by default__ with options to disable.
    - An example with the sync command: `rclone sync <data source>  <configured_name>:waylab-durbinlab-bucket`
+
+## Frequently Asked Questions (FAQ)
+
+- __Question__: What should I do if I see the error: `"Error 400: Cannot insert legacy ACL for an object when uniform bucket-level access is enabled."`?
+- __Answer__: Try using the following:
+  - Add an environment variable like the following (command may vary based on operating system): `export RCLONE_GCS_BUCKET_POLICY_ONLY=true`
+  - Use an additional command-line flag `rclone --gcs-bucket-policy-only ...<the rest of your command goes here>...`
```

Comment:
My understanding from the docs (https://rclone.org/googlecloudstorage/#gcs-bucket-policy-only) is that you just need one of those options: specifying `--gcs-bucket-policy-only` is the same as setting `RCLONE_GCS_BUCKET_POLICY_ONLY=true` and as far as I can tell the flag overrides the env var. Maybe saying something like "Try using one of the following options" would be clearer?

Reply:
Thanks, great point, I'll get this updated!